api/provider/aws/load_balancer_update.go

```diff
@@ -0,0 +1,51 @@
+package aws
+
+import (
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/quintilesims/layer0/common/models"
+)
+
+func (l *LoadBalancerProvider) Update(req models.UpdateLoadBalancerRequest) (*models.LoadBalancer, error) {
+	ports := req.Ports
```

Comment:
Looks like you forgot to delete the body after copy+paste here?

Reply:
No, this was an attempt to actually implement the function. I think I used [Read()](https://github.com/quintilesims/layer0/blob/274-scheduler-skeleton-jparsons/api/provider/aws/load_balancer_read.go#L13) as the basic structure for attempting to implement it

Reply:
Implementing the `provider` layer is kind of a beast on its own. I think bivers is trying to keep the implementation of these layers as discrete tasks.